### PR TITLE
panflute recipe

### PR DESCRIPTION
**Pull request by kiwi0fruit**

### CODE REVIEW

**Comment by kiwi0fruit** [comment]
@conda-forge/help-python please review.

**Comment by kiwi0fruit** [comment]
@conda-forge-admin @conda-forge/help-python please review.

File: recipes/panflute/LICENSE [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2016, Sergio Correia
```
Comment (beckermr) [inline recipes/panflute/LICENSE]: Please make an issue/PR upstream to make sure the license is included in the pypi tarball.

Comment (kiwi0fruit) [reply recipes/panflute/LICENSE]: but why?

Comment (beckermr) [reply recipes/panflute/LICENSE]: Maintenance and the license probably requires it for the pypi dist. Not a big deal and this recipe is merged.

Comment (beckermr) [reply recipes/panflute/LICENSE]: Once it is in the pypi tarballs, then you simply put the name of the file. Conda build will find it.

**Pull request merged**